spec/use-hotkeys/use_log_controller.js

```diff
@@ -0,0 +1,22 @@
+import { Controller } from 'stimulus'
+import { bindHotkeys, unbindHotkeys } from '../../src'
+
+export default class UseLogController extends Controller {
+  static hotkeys = { '/': 'singleKeyHandler', 'cmd+a': 'metaKeyHandler' }
+
+  connect() {
+    bindHotkeys(this)
+  }
+
+  disconnect() {
+    unbindHotkeys(this)
```

Comment:
Does the developer really need to call this manually? Can't we somehow attach a callback/eventlistener on document or somewhere to call this function accordingly?

Reply:
If you know of any way to do this, that would be 👨‍🍳 😘 ... I currently do not. That would also solve the naming problem above.

@adrienpoly ?

Reply:
Do stimulus controller emit an event on disconnect that I could use here? But then again, that's another event listener 🤔

Reply:
Ah, I'll take a look at use-hover for example, that looks sane 👍🏻